generators/python-v2/ast/src/__test__/__snapshots__/Class.test.ts.snap

```diff
@@ -13,6 +13,13 @@ class MyDataClass:
 "
 `;
 
+exports[`class > class with de-indented multi-line string 1`] = `
+"class MyClass:
+    multiline_string: str = """Hello
+    World"""
+"
```

Comment:
I would expect for `writer.writeNoIndent` to not indent this line. There should be no whitespace before `World` here